src/pages/MonthlyGood/MonthlyGoodPage.vue

```diff
@@ -1,47 +1,199 @@
+						Changes to this contribution can be made in your
+						<a href="https://www.kiva.org/settings/credit">credit settings</a>.
+					</h4>
+				</div>
+			</template>
+		</kiva-as-expert>
+		<more-about-kiva />
+		<frequently-asked-questions />
 	</www-page>
 </template>
 
 <script>
+import _get from 'lodash/get';
+import gql from 'graphql-tag';
+
 import WwwPage from '@/components/WwwFrame/WwwPage';
+import KvHero from '@/components/Kv/KvHero';
+import KvResponsiveImage from '@/components/Kv/KvResponsiveImage';
+
 import LandingForm from './LandingForm';
+import HowItWorks from './HowItWorks';
+import EmailPreview from './EmailPreview';
+import MoreAboutKiva from './MoreAboutKiva';
+import KivaAsExpert from './KivaAsExpert';
+import FrequentlyAskedQuestions from './FrequentlyAskedQuestions';
+
+const pageQuery = gql`{
+	my {
+		autoDeposit {
+			isSubscriber
+		}
+	}
+}`;
+
+const heroImagesRequire = require.context('@/assets/images/mg-landing-hero', true);
 
 export default {
+	components: {
+		WwwPage,
+		LandingForm,
+		KvHero,
+		KvResponsiveImage,
+		HowItWorks,
+		EmailPreview,
+		MoreAboutKiva,
+		KivaAsExpert,
+		FrequentlyAskedQuestions
+	},
 	data() {
 		return {
+			isMonthlyGoodSubscriber: false,
 			monthlyGoodAmount: 25,
 			selectedCategory: 'default',
+			heroImages: [
+				['small', heroImagesRequire('./monthlygood-banner-sm-std.jpg')],
+				['small retina', heroImagesRequire('./monthlygood-banner-sm-retina.jpg')],
+				['medium', heroImagesRequire('./monthlygood-banner-med-std_0.jpg')],
+				['medium retina', heroImagesRequire('./monthlygood-banner-med-retina_0.jpg')],
+				['large', heroImagesRequire('./monthlygood-banner-lg-std_0.jpg')],
+				['large retina', heroImagesRequire('./monthlygood-banner-lg-retina_0.jpg')],
+				['xga', heroImagesRequire('./monthlygood-banner-xl-std_0.jpg')],
+				['xga retina', heroImagesRequire('./monthlygood-banner-xl-retina_0.jpg')],
+				['wxga', heroImagesRequire('./monthlygood-banner-xxl-std.jpg')],
+				['wxga retina', heroImagesRequire('./monthlygood-banner-xxl-retina.jpg')],
+			],
 		};
 	},
-	components: {
-		WwwPage,
-		LandingForm
-	}
+	inject: ['apollo'],
+	apollo: {
+		query: pageQuery,
+		preFetch: true,
+		result({ data }) {
+			this.isMonthlyGoodSubscriber = _get(data, 'my.autoDeposit.isSubscriber', false);
+		},
+	},
+
 };
 
 </script>
 
 <style lang="scss" scoped>
 @import 'settings';
 
+.already-subscribed-msg-wrapper {
+	background-color: $vivid-yellow;
+	padding: 0.625rem;
+	margin-top: 1.25rem;
+}
+
+.mg-hero {
+	margin-bottom: 0;
+
+	::v-deep form {
+		// overwrite styles for error display over hero image
+		.validation-errors {
+			border: 1px solid $charcoal;
+			background-color: rgba(255, 255, 255, 0.7);
+			padding: 0.15rem 0 0 0.45rem;
+			line-height: 1.25rem;
+			width: 100%;
+		}
+	}
+
+	//set min height to improve sizing when image has not loaded yet
+	min-height: 100px;
+	@media only screen and (min-width: 761px) {
+		min-height: 355px;
```

Comment:
min-height: 22.19rem;

Reply:
For these more complex ones you can also use rem-calc e.g. `rem-calc(355)`